settings/LoanPolicyDetail.js

```diff
@@ -2,9 +2,17 @@ import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+
 import { stripesShape } from '@folio/stripes/core';
-import { Accordion, Col, ExpandAllButton, KeyValue, Row } from '@folio/stripes/components';
+import {
+  Accordion,
+  ExpandAllButton,
+  KeyValue,
+  Col,
+  Row,
+} from '@folio/stripes/components';
 import { ViewMetaData } from '@folio/stripes/smart-components';
```

Comment:
There should be an empty line after folio imports in order to separate them from own imports
```suggestion
import { ViewMetaData } from '@folio/stripes/smart-components';

```

Reply:
Both the imports are from the @folio

Reply:
Isn't PolicyPropertyDescription on line 15 is your own import?

Reply:
Apologies, missed that